scss/sections/trade/_trade_respo.scss

```diff
@@ -97,18 +134,42 @@
 
 @media (min-width: 1200px) {
     .trade {
-        .trade_header {
+        .navyblue_section {
             p {
                 font-size: 44px;
             }
         }
         .calculator_container {
-            .container {
-                .left_input {
-                    .amount_of_currency {
-                        width: 40%;
+            .top_calculator_part {
+                display: flex;
+                flex-wrap: nowrap;
+                flex-direction: row;
+                width: 100%;
+                height: 60%;
+                .first_item_container {
```

Comment:
bad naming if you want to use first sec last in selectors just use child selectors in css